controllers/goroutines/operator_status.go

```diff
@@ -82,37 +83,62 @@ func UpdateCsCrStatus(bs *bootstrap.Bootstrap) {
 	}
 }
 
-func getBedrockOperator(bs *bootstrap.Bootstrap, name, namespace string) apiv3.BedrockOperator {
+func getBedrockOperator(bs *bootstrap.Bootstrap, name, namespace string) (apiv3.BedrockOperator, error) {
 	var opt apiv3.BedrockOperator
+	opt.Name = name
 
 	// fetch subscription
 	sub := &olmv1alpha1.Subscription{}
 	subKey := types.NamespacedName{
 		Name:      name,
 		Namespace: namespace,
 	}
-	if subErr := bs.Client.Get(ctx, subKey, sub); subErr != nil {
-		return opt
+	if err := bs.Client.Get(ctx, subKey, sub); err != nil {
+		return opt, err
 	}
-	installCSV := sub.Status.InstalledCSV
-	if installCSV != "" {
-		opt.Name = installCSV[:strings.IndexByte(installCSV, '.')]
-		opt.Version = installCSV[strings.IndexByte(installCSV, '.')+1:]
+	installedCSV := sub.Status.InstalledCSV
+	if installedCSV != "" {
+		opt.Name = installedCSV[:strings.IndexByte(installedCSV, '.')]
+		opt.Version = installedCSV[strings.IndexByte(installedCSV, '.')+1:]
 	}
 
 	// fetch csv
 	csv := &olmv1alpha1.ClusterServiceVersion{}
 	csvKey := types.NamespacedName{
-		Name:      installCSV,
+		Name:      installedCSV,
 		Namespace: namespace,
 	}
-	if csvErr := bs.Reader.Get(ctx, csvKey, csv); csvErr != nil {
-		return opt
+	if err := bs.Reader.Get(ctx, csvKey, csv); err != nil {
+		klog.Warningf("%s CSV %s", name, err)
```

Comment:
Also a little advice here, we could use `klog.Warningf("Failed to get %s CSV: %s", name, err)` to be consistent with other error msg.